workflow-templates/im-deploy-windows-service.yml

```diff
@@ -209,7 +146,11 @@ jobs:
   # Details on how to use the action can be found in the action's README.
 
   deploy-service:
-    needs: [set-vars, stakeholder-approval, validate-tag-for-prod-deploys]
+    needs: |
+      - verify-tag
+      - stakeholder-approval
+      - attestor-approval
+      - validate-tag-for-prod-deploys
```

Comment:
Should this be `validate-tag-is-in-main-for-prod-deploys`?

Reply:
Yes, it should. 👍🏽